specification/app/resource-manager/Microsoft.App/stable/2022-03-01/SourceControls.json

```diff
@@ -177,13 +177,19 @@
         ],
         "responses": {
           "200": {
-            "description": "No change performed on the Container App.",
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SourceControl"
+            }
+          },
+          "201": {
+            "description": "Created",
             "schema": {
               "$ref": "#/definitions/SourceControl"
             }
           },
           "202": {
-            "description": "Container App create or update has been started.",
+            "description": "Accepted",
```

Comment:
We can add a more descriptive description for this, maybe: `Source Controls integration accepted`